code/game/objects/structures/extinguisher.dm

```diff
@@ -68,7 +68,7 @@
 	return
 
 /obj/structure/extinguisher_cabinet/AltClick(mob/living/user)
-	if(user.incapacitated() || !Adjacent(user))
+	if(user.incapacitated() || !Adjacent(user) || !ishuman(user))

```

Comment:
we have isliving(), don't we? (or if we don't, why not?)

I'm not sure if calling isliving() is any more or less expensive than !istype(). I'm going to guess that it probably is more or less the same thing.